### PR TITLE
[2.3] Updates for v1.16.12-rancher1-1 & v1.17.8-rancher1-1

### PR DESCRIPTION
**Pull request by ryansann**
- add new templates
- change default
- generated data

### CODE REVIEW

File: rke/k8s_version_info.go [hunk 1]
```diff
@@ -131,6 +131,12 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
 			MinRKEVersion:     "1.0.9-rc0",
 		},
 		// The Calico/Canal template in this version use functions that are only available in RKE v1.0.0 and up
+		// This version includes nodelocal dns only available in RKE v1.0.7 and up
+		"v1.16.12-rancher1-1": {
+			MinRancherVersion: "2.3.8-rc0",
+			MinRKEVersion:     "1.0.9-rc0",
```
Comment (maggieliu) [inline rke/k8s_version_info.go:137]: RKE min version should be 1.0.11

File: rke/k8s_version_info.go [hunk 2]
```diff
@@ -159,6 +165,12 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
 			MinRancherVersion: "2.3.8-rc0",
 			MinRKEVersion:     "1.0.9-rc0",
 		},
+		// The Calico/Canal template in this version use functions that are only available in RKE v1.0.0 and up
+		// This version includes nodelocal dns only available in RKE v1.0.7 and up
+		"v1.17.8-rancher1-1": {
+			MinRancherVersion: "2.3.8-rc0",
+			MinRKEVersion:     "1.0.9-rc0",
```
Comment (maggieliu) [inline rke/k8s_version_info.go:172]: RKE min version should be 1.0.11

File: rke/k8s_rke_system_images.go [hunk 3]
```diff
@@ -2509,6 +2509,41 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  m("rancher/kubelet-pause:v0.1.4"),
 			Nodelocal:                 m("k8s.gcr.io/k8s-dns-node-cache:1.15.7"),
 		},
+		// Enabled out of band for Rancher v2.3.8 with Rancher v2.4.5
```
Comment (maggieliu) [inline rke/k8s_rke_system_images.go:2512]: Enabled out of band for Rancher v2.3.8 with Rancher v2.4.4

File: rke/k8s_rke_system_images.go [hunk 4]
```diff
@@ -2843,6 +2878,41 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  m("rancher/kubelet-pause:v0.1.4"),
 			Nodelocal:                 m("k8s.gcr.io/k8s-dns-node-cache:1.15.7"),
 		},
+		// Enabled in Rancher v2.3.8
```
Comment (maggieliu) [inline rke/k8s_rke_system_images.go:2881]: Enabled out of band for Rancher v2.3.8 with Rancher v2.4.5

Comment (deniseschannon) [reply rke/k8s_rke_system_images.go:2881]: This still isnt updated

**Comment by deniseschannon** [comment]
This can be closed, right?

**Pull request closed without merging**